Cargo.toml

```diff
@@ -1,14 +1,18 @@
 [package]
-
 name = "rustfmt-nightly"
 version = "1.5.1"
-description = "Tool to find and fix Rust formatting issues"
-repository = "https://github.com/rust-lang/rustfmt"
-readme = "README.md"
-license = "Apache-2.0/MIT"
 build = "build.rs"
 categories = ["development-tools"]
 edition = "2021"
+license = "Apache-2.0/MIT"
+readme = "README.md"
+repository = "https://github.com/rust-lang/rustfmt"
+description = "Tool to find and fix Rust formatting issues"
+
+# Rustc dependencies are loaded from the sysroot, Cargo doesn't know about them.
+[package.metadata.rust-analyzer]
+# This package uses #[feature(rustc_private)]
+rustc_private = true
```

Comment:
I wonder if it's better to hold off on formatting rustfmt's Cargo.toml until after this feature is added. I feel like it might be better to focus on implementing the feature in the PR and then we can always follow up with a formatting change in a follow up PR.